### ISSUE TITLE
Ghost tabs when closing some tabs

### ISSUE THREAD

**Issue by Trollwut** [piroor/treestyletab#1117]
While using the addon "Tree Style Tabs" I have a strange behaviour.

Do you see the spaces pointed out in this picture?

They have a height of about 1-3 px and well, are just spaces. But when clicking on them, it seems like an old abandones closed tab is still there as the space. See attached picture for clarification.
![h_1461252136_2205616_77f3df9a15](https://cloud.githubusercontent.com/assets/3462975/15111763/87c34c40-15e9-11e6-9ea8-d5373fb715f3.png)

For example on this screenshot I clicked the last space (right above the + button) and as you see my active page is about:addons, which I had closed before.

The spaces don't vanish. When on a ghost tab, I also can normally use the website. So as it's working, it will also drain performance.

They seem to appear randomly: No specific websites, but sometimes when I close a tab, it transform into a space.

I can't just click and CTRL+W them - this shortcut doesn't work. (But I can open child tabs from it)

So how can I get rid of them?
## Short description

Sometimes when closing a tab, it will create a "ghost tab". It creates a 1px line and when clicking on it, the closed tab is shown again. But you can't close it again/forever.
Upon restarting Firefox, the tab is restored into a normal tab, but closing it again will result in another ghost tab.
## Steps to reproduce

I'm really not sure what chooses a tab to get a ghost tab. I only know that after restarting, the former ghost tab is again a normal tab. But closing it again will create a ghost tab again.
## Expected result

Closing the tab should just close the tab. Eliminate it.
## Actual result

Ghost tabs.
## Environment
- Platform (OS): Linux
- Version of Firefox: 46.0.1
- Version (or revision) of Tree Style Tab: 0.17.2016031101

**Comment by piroor** [comment]
Could you shrink the list of other addons to reproduce the problem? I think that Stylish or someone can conflict to TST.

**Comment by Trollwut** [comment]
Unfortunately not, the problem still exists. But I have also switched to the Developer version of Firefox, which does not have my problem. So I guess an upcomming update will fix it.

I have also found out a new behaviour: It's always the first two/three tabs that are closed after opening Firefox which will become ghost tabs.

I can open about:performance and close ghost tabs with that, but if I close one ghost tab, the next tab to be closed will become one for it.
